exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
```

Comment:
This code is correct and clear. However, you are able to simplify the logic here by deleting some conditions in the if statements. Think about the order of the if statements and what you are checking in each if statement.